spec/models/product_spec.rb

```diff
@@ -232,7 +209,7 @@ def requires_account?
       end
     end
 
-    context "can_purchase?" do
+    context "can_purchase?", feature_setting: { user_based_price_groups: true } do
```

Comment:
Probably something for @meara to weigh in on, she was the person who touched this last.